focus-android/app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -71,6 +75,15 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
     private lateinit var startupTypeTelemetry: StartupTypeTelemetry
     private var _binding: ActivityMainBinding? = null
     private val binding get() = _binding!!
+    private lateinit var privateNotificationObserver: PrivateNotificationFeature
```

Comment:
Could we initialize this here and convert this to val? And call `start()` in `onCreate()`

Reply:
We need context as a param here, so initializing it here would throw an NPE.